File: 04-07-2025/RazorPayIntegration/AngularApp/src/app/features/order/models/order.ts
import { PaymentModel } from "../../payment/models/payment";

export class OrderModel {
  constructor(
    public id: string,
    public customerName?: string,
    public email?: string,
    public contactNumber?: string,
    public razorpayOrderId?: string,
    public payment?: PaymentModel
  ) {}
}

File: 04-07-2025/RazorPayIntegration/DotnetAPI/Models/Order.cs
namespace DotnetAPI.Models
{
    public class Order
    {
        public Guid Id { get; set; }
        public string CustomerName { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string ContactNumber { get; set; } = string.Empty;
        public string RazorpayOrderId { get; set; } = string.Empty;
        public Payment? Payment { get; set; }
        public Guid? PaymentId { get; set; }
    }
}